fix(security): bge-m3 missing read_only in compose.vps.yml

## Problem

Тест `test_vps_service_has_read_only[bge-m3]` падает:

```
AssertionError: vps:bge-m3 must have read_only: true (matching dev compose)
```

## Root Cause

`compose.vps.yml` не устанавливает `read_only: true` для сервиса `bge-m3`, хотя dev compose это делает.

## Fix

Добавить `read_only: true` в `compose.vps.yml` для `bge-m3`:

```yaml
bge-m3:
  read_only: true
```

## Affected Tests

- `test_compose_config.py::TestVpsSecurityBaseline::test_vps_service_has_read_only[bge-m3]`

--- a/compose.vps.yml
+++ b/compose.vps.yml
@@ -2,6 +2,11 @@
 # Usage: COMPOSE_FILE=compose.yml:compose.vps.yml docker compose up -d
 
 services:
+  bge-m3:
+    read_only: true
+    tmpfs:
+      - /tmp
+
   bot:
     environment:
       RERANK_PROVIDER: "none"

